yt/frontends/ramses/particle_handlers.py

```diff
@@ -351,7 +351,7 @@ def read_header(self):
             fields = list(self.known_fields)
 
         for i in range(self.ds.dimensionality * 2 + 1):
-            for j in range(self.ds.max_level, self.ds.min_level):
+            for j in range(self.ds.min_level, self.ds.max_level):
```

Comment:
weird. This is actually an unrelated bugfix, right ?

Reply:
It is, but it seems it was never tested...